packages/react/react/src/components/SwitchCase/index.test.tsx

```diff
@@ -14,20 +14,21 @@ describe('SwitchCase', () => {
       />
     );
   }
-  it(`'a' 라는 값을 주면 'Hello A' 가 렌더링 된다.`, () => {
+
+  it(`When the value is 'a', it should render 'Hello A'.`, () => {
```

Comment:
```suggestion
  it(`should render 'Hello A' When the value is 'a'.`, () => {
```

you can refer [this comment](https://github.com/toss/slash/pull/267#discussion_r1248051773)

Reply:
Thank You!!!